numpy/core/umath.py

```diff
@@ -10,8 +10,7 @@
 from ._multiarray_umath import *  # noqa: F403
 # These imports are needed for backward compatibility,
 # do not change them. issue gh-11862
-# _ones_like is semi-public, on purpose not added to __all__
-from ._multiarray_umath import _UFUNC_API, _add_newdoc_ufunc, _ones_like
+from ._multiarray_umath import _UFUNC_API, _add_newdoc_ufunc
```

Comment:
The comment above specifically asks not to change them.  So a cleanup is not good enough, you can add a comment  if that helps with checkers reporting unused imports.